locale/en/docs/guides/simple-profiling.md

```diff
@@ -63,7 +63,8 @@ app.get('/auth', (req, res) => {
     return res.sendStatus(400);
   }
 
-  const hash = crypto.pbkdf2Sync(password, users[username].salt, 10000, 512);
+  const salt = users[username].salt;
```

Comment:
Nit: Use destructuring.

Reply:
Sorry for mistake, and I'll fix that sooooooooon :)